src/Microsoft.AspNet.Http/Authentication/DefaultAuthenticationManager.cs

```diff
@@ -13,14 +13,19 @@
 
 namespace Microsoft.AspNet.Http.Authentication.Internal
 {
-    public class DefaultAuthenticationManager : AuthenticationManager
+    public class DefaultAuthenticationManager : AuthenticationManager, IDisposable
```

Comment:
I'm not a fan of using IDisposable to release memory for the pooled object, feels like mismatch...